plugins/foreman_discovery/2.0/index.md

```diff
@@ -341,7 +341,17 @@ This should by done automatically by our installer. It is a good idea to check
 if the host responds properly and there are no firewalls blocking the
 communication.
 
-## 3.3 Permissions
+## 3.3 Subnet proxy setup
+
+All subnets containing discovered nodes need to be specified to allow Foreman
```

Comment:
Ah, context always helps.  Let me suggest a more accurate rewrite:

"All subnets with discovered nodes need this specified in Foreman so it connects via the Foreman Proxy."